infra/cifuzz/build_fuzzers_entrypoint.py

```diff
@@ -46,7 +46,7 @@ def build_fuzzers_entrypoint():
   if not build_fuzzers.build_fuzzers(config):
     logging.error(
         'Error building fuzzers for project %s (commit: %s, pr_ref: %s).',
```

Comment:
I think most users know what project they are using, maybe take out 'project %s' from this log line entirely.